src/utils.js

```diff
@@ -343,3 +348,31 @@ const sleep = (millis) => {
 export const publicUtils = {
     sleep,
 };
+
+/**
```

Comment:
I would export this function in `Apify.utils` which means to include this in `export const publicUtils` and add `  * @memberof utils` to docs